src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -71,9 +183,12 @@ export class ConfirmBid extends React.Component<ConfirmBidProps> {
               You agree to <LinkText onPress={this.onPressConditionsOfSale}>Conditions of Sale</LinkText>.
             </Serif14>
 
-            <Flex m={4}>
-              <Button text="Place Bid" onPress={() => null} />
-            </Flex>
+            <Button
+              text="Place Bid"
+              onPress={() => {
```

Comment:
I'm curious, since I'm still getting oriented around JS, could this be rewritten as `onPress={this.placeBid}`? Or is there some JS binding problem?

Reply:
I think `onPress={() => this.placeBid()}` is similar to `onPress={this.placeBid.bind(this)}` (to bind `this` back to the react component object instead of event handler's `this`) but there has been ongoing debate about what is the best way to do that. I haven't seen any objection to `onPress={() => this.placeBid()}` recently but doing `bind` on each render is not great 😆